consensus/consensus_fbft_test.go

```diff
@@ -0,0 +1,15 @@
+package consensus
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestLockedFBFTPhase(t *testing.T) {
+	s := NewLockedFBFTPhase(FBFTAnnounce)
+	require.Equal(t, FBFTAnnounce, s.Get())
+
+	s.Set(FBFTCommit)
+	require.Equal(t, FBFTCommit, s.Get())
+}
```

Comment:
It might be good if we test it by multiple threads rather than a normal Set and Get.

Reply:
We have
`
LockAcquire
`
`
Assign/Retrieve
`
`
LockRelease
`
What should be tested here by concurrent test? How we should prove that it was really concurrent?

Reply:
Besides this concurrent test, we have working code with no race errors. Is it not enough?